hal/ubuntu/qemu.sh

```
@@ -14,9 +14,11 @@ RAM="16384M"
 CPU="8"
 USER="ultraviolet"
 PASSWORD="password"
```

Shellcheck (0.11.0)

[warning] PASSWORD appears unused. Verify use (or export if used externally).

(SC2034)